src/gateway/index.md

```diff
@@ -3,6 +3,39 @@ title: Kong Gateway
 subtitle: API gateway built for hybrid and multi-cloud, optimized for microservices and distributed architectures
 ---
 
+## Quick Links
+
+<div class="docs-grid-install">
+
+  <a href="#features" class="docs-grid-install-block no-description">
+    <img class="install-icon" src="/assets/images/icons/documentation/icn-flag.svg" alt="">
```

Comment:
```suggestion
    <img class="install-icon no-image-expand" src="/assets/images/icons/documentation/icn-flag.svg" alt="">
```
Need to add a class so this doesn't happen:
![Screen Shot 2022-09-06 at 10 36 14 AM](https://user-images.githubusercontent.com/54370747/188702494-80deb2d4-d193-4f05-b190-0feb4fea771a.png)

It should really go the other way - add a class when you need to expand, not have it be the default. But for now, this is what we've got.